test/tidy.lean

```diff
@@ -13,10 +13,7 @@ meta def interactive_simp := `[simp]
 
 def tidy_test_0 : ∀ x : unit, x = unit.star :=
 begin
-  success_if_fail { chain [ interactive_simp ] },
-  intro1,
-  induction x,
-  refl
+  simp
```

Comment:
Should this test be testing something about `tidy` in some way? @semorrison

Reply:
It had been a test about `chain`, checking that it failed when it was meant to fail. The test has become a bad one, because now the underlying tactic is succeeding (yay!). I propose we just delete this test entirely.